src/users/userSelectors.js

```diff
@@ -29,6 +30,13 @@ export const getAccountDetailsUser = createSelector(
   },
 );
 
+export const getUserByEmail = (users: any[], userEmail: string) =>
```

Comment:
I have created this function in multiple PR's so lets have this, as it will be used in many places!